public_html/wp-content/themes/pattern-directory/css/components/_site-header.scss

```diff
@@ -0,0 +1,88 @@
+body.home {
+	.site-header {
+		padding: 3.75rem 1rem;
+		text-align: left;
+	}
+
+	.site-branding {
+		display: grid;
+		grid-template-areas:
+			"title   callout"
+			"tagline callout";
+		grid-template-rows: auto 1fr;
+		grid-template-columns: 1fr 18rem;
```

Comment:
IE 11 doesn't support grid-template-* without prefixes, I believe. I think we have a prefixer configured but it's worth a manual check using IE 11 if you haven't already.